gradle/libs.versions.toml

```diff
@@ -20,7 +20,7 @@ hiltNavigation = "1.2.0"
 biometricKtx = "1.2.0-alpha05"
 coilCompose = "2.6.0"
 kotlinxSerialization = "1.6.2"
-sargon = "1.0.31-598a8a45"
+sargon = "1.1.0-host_info-777f0b3d"
```

Comment:
This will be replaced to the actual build from main branch when sargon has `sargon_os` merged.